cfgov/v1/models/base.py

```diff
@@ -220,6 +240,9 @@ def get_context(self, request, *args, **kwargs):
                 path=Value(request.path, output_field=models.CharField())) \
             .filter(path__regex=F('url_pattern'))
 
+        if self.schema_json:
```

Comment:
![image](https://user-images.githubusercontent.com/10562538/79778852-645c4600-8307-11ea-8760-6bf4f02205af.png)
You'll probably get a test coverage failure because this conditional isn't covered.

Reply:
> # Todos
> - [ ] Add tests for new context variable

😜